### PR TITLE
Adds dispatch feature

### PR DESCRIPTION
**Pull request by ddbruce**

### CODE REVIEW

File: public/js/whiteboard.js [hunk 1]
```diff
@@ -1,137 +1,233 @@
 /* global moment, io, SunCalc */
 
-const longitude = -73.675770;
-const latitude = 42.729270;
+const longitude = -73.67577;
+const latitude = 42.72927;
```
Comment (ddbruce) [inline public/js/whiteboard.js:4]: These should get pulled out into environment variables

File: server.js [hunk 2]
```diff
@@ -93,7 +94,17 @@ app.get('/mishap', async (_, res) => {
...
     io.emit('chores', req.body);
-    res.send({success: true});
+    res.send({ success: true });
+});
+
+app.post('/dispatch', (req, res) => {
+    if (HERALD_TOKEN !== req.query.token) {
+        res.sendStatus(403);
+    } else {
+        console.log('recieved herald dispatch');
```
Comment (ddbruce) [inline server.js:104]: Spellcheck

File: src/crews.js [hunk 3]
```diff
@@ -1,6 +1,8 @@
 const { execQuery } = require('./db');
 const moment = require('moment');
 
+const DEV = process.env.ENV == 'dev' ? true : false;
```
Comment (ddbruce) [inline src/crews.js:4]: This can be rewritten `const DEV = process.env.ENV == 'dev';` because that itself returns `true` or `false`.

**Comment by ddbruce** [comment]
@CKegel see the above. I can't explicitly request feedback because I created the PR.

File: public/js/whiteboard.js (hunk 1 above)
Comment (CKegel) [reply public/js/whiteboard.js:4]: How would one do this on the client side?

Comment (ddbruce) [reply public/js/whiteboard.js:4]: Ermmm you wouldn't. My mistake. This should probably be pulled from the server via API which will have it stored in an env var. Alternately, we could use the [HTML Geo API](https://www.w3schools.com/html/html5_geolocation.asp), but this is probably less desirable.

Comment (CKegel) [reply public/js/whiteboard.js:4]: https://github.com/techinems/headsup/pull/71

**Review by lramos15 — DISMISSED** [review]

**Comment by ddbruce** [comment]
Approved with only a housekeeping `README` update afterwards; I'm going to merge by bypassing protections.

**Comment by ddbruce** [comment]
Thanks to @CKegel for putting in basically all of the legwork for this!

**Pull request merged**